Resources/translations/sub_items.en.xliff

```diff
@@ -32,18 +37,23 @@
         <note>key: bulk_delete.success.message</note>
       </trans-unit>
       <trans-unit id="364a3498e021163497d9110380736b876beb326f" resname="bulk_move.error.message">
-        <source>You do not have permission to move at least 1 of the selected content item(s). Please contact your Administrator to obtain permissions.</source>
-        <target state="new">You do not have permission to move at least 1 of the selected content item(s). Please contact your Administrator to obtain permissions.</target>
+        <source><![CDATA[%notMovedCount% of the %totalCount% selected item(s) could not be moved due to lack of your user permissions. <u><a class='ez-notification-btn ez-notification-btn--show-modal'>Click here for more information.</a></u><br>Please contact your Administrator to obtain permissions.]]></source>
+        <target state="new"><![CDATA[%notMovedCount% of the %totalCount% selected item(s) could not be moved due to lack of your user permissions. <u><a class='ez-notification-btn ez-notification-btn--show-modal'>Click here for more information.</a></u><br>Please contact your Administrator to obtain permissions.]]></target>
         <note>key: bulk_move.error.message</note>
       </trans-unit>
+      <trans-unit id="44a971e3ea91b95b9fbbd596f54ca632ef0865d4" resname="bulk_move.error.modal.table_title">
+        <source>Content items cannot be moved (%itemsCount%)</source>
```

Comment:
`(%itemsCount%) Content item(s) cannot be moved`? Not sure if that is the context

Reply:
The idea is that the main part is: `Content item(s) cannot be moved` and at the end we show number of items in brackets, so e.g.: `Content item(s) cannot be moved (12)`. We have similar approaches in other places, in such as: `Bookmarks (12)`, or `Sub-items (12)`.
Here we have bigger title but I think it is okey. 🙂